tools/go.mod

```
@@ -1,10 +1,13 @@
 module go.opentelemetry.io/contrib/tools
 
-go 1.20
+go 1.21
+
+toolchain go1.21.1
```

Is the intention of this line to use the newest compiler? 

If so then we could

```suggestion
toolchain go1.21.2
```